<launch>

	<arg name="arduino_port" default="/dev/ttyACM0" />
	<arg name="robot_name" default="asl_gremlin1" />

	                    <!-- Launch MAVROS -->
    <include file="$(find mavros)/launch/apm2.launch" 
        ns="$(arg robot_name)" />
        
                        <!-- Launch ROSSERIAL -->
    <node ns="$(arg robot_name)" name="rosserial" pkg="rosserial_python" type="serial_node.py" respawn="true">
    <param name="port" value="$(arg arduino_port)" /> 
	</node>

</launch>